Packs/CTIX/Integrations/CTIX/CTIX.yml

```diff
@@ -0,0 +1,1137 @@
+description: This is Cyware Threat Intelligence eXhange(CTIX) integration which enriches
+  IP/Domain/URL/File Data.
+detaileddescription: |
+  1. Navigate to Administration > Integrations management > CTIX Integrators.
+  2. Click “Add New” to create and configure a new integration instance and download the key files.
+  3. Login to the Cortex XSOAR application and go to the App Configuration page.
+  4. Fill in the following details.
+      a. Name: Give a name for the integration instance.
+      b. Endpoint URL: Enter the endpoint URL of your CTIX Instance. The Endpoint URL can be generated by a licensed user from the CTIX application. For example, https://<domain_name>/ctixapi/openapi/
+      c. Access Key: Enter the Access Key from the CTIX application. The Access Key can be generated by a licensed user from the CTIX application. For example, “74xxxxx7-xxxb-4xxa-xxxx-0xxxxxxxxxx2“.
+      d. Secret Key: Enter the Secret Key from the CTIX application. The Secret Key can be generated by a licensed user from the CTIX application. For example, “0xxxxxx8-xxxx-4xx4-xxx6-5xxxxxxxxxxc”.
+  5.  After finishing, click the “Test“ button to validate the URL, Token, and Connection.
+configuration:
+- display: Endpoint URL
+  name: base_url
+  defaultvalue: ""
+  type: 0
+  required: true
+  additionalinfo: Enter the endpoint URL of your CTIX Instance.
+- display: Access Key
+  name: access_id
+  defaultvalue: ""
+  type: 4
+  required: true
+  additionalinfo: Enter the Access Key from the CTIX application.
+- display: Secret Key
+  name: secret_key
+  defaultvalue: ""
+  type: 4
+  required: true
+  additionalinfo: Enter the Secret Key from the CTIX application.
+script:
```

Comment:
please add the proxy and trust any certificate params. (Required also in the demisto-sdk validates.